src/ks/hangedman/Colour.java

```diff
@@ -0,0 +1,23 @@
+package ks.hangedman;
+
+public enum Colour {
+
+    RED("\u001B[31m"),
+    GREEN ("\u001B[32m"),
```

Comment:
please reformat this to use whitespaces consistently throughout entire project. In this case, either remove spaces before parenthesis or add space for RED